setup.cfg

```diff
@@ -36,6 +36,8 @@ install_requires =
   sunpy[net,asdf]>=4.0.7
   setuptools>=59
   jsonschema>=3.2
+  aiohttp>=3.6
```

Comment:
The fact we are doing this for typehints is somewhat annoying but hey.

Reply:
We are? I was just going by NEP 29.